frontend/public/locales/en/translation.json

```
@@ -2957,6 +2956,7 @@
   "To run automation manually, select \"Disabled\" and check the \"Manual run\" checkbox.)": "To run automation manually, select \"Disabled\" and check the \"Manual run\" checkbox.)",
   "To search for a keyword, type the word in the search box.": "To search for a keyword, type the word in the search box.",
   "To search for resources with a given property value, type or select the property name from the autocomplete list. Then type or select the value for the selected property filter.": "To search for resources with a given property value, type or select the property name from the autocomplete list. Then type or select the value for the selected property filter.",
+  "To view managed virtual machines, you must enable the Red Hat Advanced Cluster Management.": "To view managed virtual machines, you must enable the Red Hat Advanced Cluster Management.",
```

Based on our convo in Slack about the user installing RHACM from the Catalog. Are users able to enable RHACM from MCE? If no, lets go with "install". 
```suggestion
  "To view managed virtual machines, you must install Red Hat Advanced Cluster Management.": "To view managed virtual machines, you must install Red Hat Advanced Cluster Management.",
```

OR 

```suggestion
  "To view managed virtual machines, you must enable Red Hat Advanced Cluster Management.": "To view managed virtual machines, you must enable Red Hat Advanced Cluster Management.",
```

---

Ok, so would it be more accurate to say, "....you must enable Search for Red Hat Advanced Cluster Management."

---

Yea, I think that best captures the error state.